src/sentry/profiles/task.py

```diff
@@ -156,6 +157,7 @@ def process_profile_task(
         return
 
     with metrics.timer("process_profile.track_outcome.accepted"):
+        _track_duration_outcome(profile=profile, project=project)
```

Comment:
Just me being extra careful here but it'd be good to wrap this in a `try ... except ...` followed by `sentry_sdk.capture_exception` in the event of some unexpected error so we at least continue emitting the current outcome